packages/test/test-end-to-end-tests/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@fluidframework/test-end-to-end-tests",
+	"name": "@fluid-internal/test-end-to-end-tests",
```

Comment:
Should we set the private flag to true?

Reply:
Actually, not sure we should. We still need this package published to our internal feeds in ADO so the test pipelines can download it from there. I know `private: true` will make it so this doesn't get published to npm, but will it prevent it from being published to our internal ADO feeds?

I see in another comment that we ran `Build -client packages` and `Build - test-tools` with these changes. We need to validate `Real Service End to End Tests`, `Real Service Stress Test`, and `Performance benchmarks` too. If those can download the package from the ADO feeds even if with `private: true`, then we're ok.

Reply:
I am getting the error from the Real Service End to End Tests pipeline; https://dev.azure.com/fluidframework/internal/_build/results?buildId=145120&view=results, We can either remove `private: true` or try to see if we can stop pulling internal packages from npm

Reply:
The pipelines always pull it from the internal feeds, not from npm; but as Matt pointed out, `private: true` makes it so `Build - cilent packages` doesn't publish things to npm _nor_ the internal feeds, so I think we do want to remove that bit. The chaneg to @fluid-internal scope should be enough to stop publishing it to npm.